models/draft.js

```diff
@@ -1,16 +1,32 @@
 let mongoose = require('mongoose');
 
+//data for each seat, human or bot
+const Seat = {
+  bot: [], //null bot value means human player
+  name: String,
+  userid: String,
+  drafted: [[]], //organized draft picks
```

Comment:
can we add schemas to all of these arrays? cards should have at minimum a cardID

Reply:
Definitely can do, and using strict schema rules here will mitigate database injection risks